### DISCUSSION TITLE
2024-06-07(금)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#108] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일

### 해야할 일
- 토큰 기반 유저 인증

### 어려운 일
- 카카오 OAuth에서 이메일과 닉네임이 안 넘어옵니다.

**Comment by devwqc** [comment]
### 완료한 일

### 해야할 일
- 헤더 + 레이아웃 최종 작업
- 홈 페이지 및 상품 목록 리스트

### 어려운 일
- 파이팅!!!!

**Comment by minyoung0503** [comment]
### 완료한 일
공동구매 컴포넌트 UI

### 해야할 일
공동구매 전체보기 페이지

### 어려운 일

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
 zzim 코드리뷰 반영
drawer
상품상세페이지

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 전체적인 결제 ui 반영
### 해야할 일
- 파일명 변경
- 결제 api..
### 어려운 일
- 결제 api 또 다른 ui 이슈
